sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -116,14 +116,23 @@ export class Receiver {
 
   /**
    * Gets an async iterator over events from the receiver.
+   *
+   * @param options Options when creating an iterator to iterate over events
+   *
+   * @returns {AsyncIterableIterator<ReceivedEventData>} AsyncIterableIterator<ReceivedEventData>
    */
   async *getEventIterator(options?: EventIteratorOptions): AsyncIterableIterator<ReceivedEventData> {
     if (!options) {
       options = {};
     }
+    let nextBatch: Promise<ReceivedEventData[]> | undefined;
+    let currentBatch = await this.receiveBatch(options.prefetchCount || 1, 60, options.cancellationToken);
```

Comment:
Minor: Can you store the `maxMessageCount` (prefetchCount) and the `maxWaitTimeInSeconds` into separate variables before the while loop? Then the existence of `prefetchCount` won't need to be performed for each `receiveBatch` call, and we avoid a [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29) for the maxWaitTimeInSeconds parameter.

Reply:
Agreed

Reply:
Updated !!